src/lib/operators/aggregate_hash.cpp

```diff
@@ -569,6 +574,40 @@ std::enable_if_t<func == AggregateFunction::Avg && !std::is_arithmetic_v<Aggrega
   Fail("Invalid aggregate");
 }
 
+// STDDEV_SAMP writes the calculated standard deviation from current aggregate and the aggregate counter
+template <typename ColumnDataType, typename AggregateType, AggregateFunction func>
+std::enable_if_t<func == AggregateFunction::StdDevSamp && std::is_arithmetic_v<AggregateType>, void>
+write_aggregate_values(std::shared_ptr<ValueSegment<AggregateType>> segment,
+                       const AggregateResults<ColumnDataType, AggregateType>& results) {
+  DebugAssert(segment->is_nullable(), "Aggregate: Output segment needs to be nullable");
+
+  auto& values = segment->values();
+  auto& null_values = segment->null_values();
+
+  values.resize(results.size());
+  null_values.resize(results.size());
+
+  size_t i = 0;
+  for (const auto& result : results) {
+    const auto count = static_cast<AggregateType>(result.aggregate_count);
+
+    if (result.current_primary_aggregate && count > 1) {
```

Comment:
so the standard deviation sample of a single sample is undefined?

Reply:
Correctly. Since the formula contains a devision by `N - 1`, using only one sample (`N = 1`) leads to a division by zero.

![image](https://user-images.githubusercontent.com/2679018/58745615-3f42b880-8453-11e9-8aa2-7605ab8a4c4e.png)